test/integration_test.rb

```diff
@@ -67,6 +67,15 @@ def test_isolated_with_fail_fast
         ENV.delete('ISOLATED')
         ENV.delete('FAIL_FAST')
       end
+
+      def test_select_untracked_files
+        FileUtils.mkdir('test')
+        fizz_test = './fizz_test.rb'
+        File.write(fizz_test, File.read(fizz_test).gsub("require_relative '\.", "require_relative '.."))
+        FileUtils.mv('./fizz_test.rb', './test/fizz_test.rb')
```

Comment:
Oops, careless me :disappointed: Thank you for noticing!
Fixed: https://github.com/Genki-S/ttnt/commit/f7ec6abb93ae7b642ed022b337ae34558dee4380